org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/services/LanguageServer.java

```diff
@@ -32,7 +32,7 @@
 	 * the server.
 	 * 
 	 * If the server receives request or notification before the initialize request it should act as follows:
-	 * 	- for a request the respond should be errored with code: -32001. The message can be picked by the server.
+	 * 	- for a request the respond should be errored with code: -32002. The message can be picked by the server.
```

Comment:
Thanks for the catch! Just a couple of suggestions for potential enhancements:
- There is the corresponding error code constant, `ResponseErrorCode.serverNotInitialized`, which could be linked to in the javadoc instead of embedding the value.
- There is another typo in this sentence: `respond` -> `response`.

Reply:
I agree. It should be `{@link org.eclipse.lsp4j.jsonrpc.messages.ResponseErrorCode#serverNotInitialized}` instead. Also, it should be "If the server receives **_a_** request or notification...". There are going to be many typos like this because this was copied from an old version of [the spec](https://microsoft.github.io/language-server-protocol/specifications/specification-current/), which often needs to be corrected because the primary maintainer does not have strong English skills.